### PR TITLE
Add naive test environment

### PR DESCRIPTION
**Pull request by mellenburg**
Helps facilitate on-prem debugging

### CODE REVIEW

**Comment by mellenburg** [comment]
@alberts here is a script that does some naive arg guessing and leave the user to invoke py.test

**Comment by mellenburg** [comment]
@cmaloney this is ready for review

File: packages/dcos-integration-test/extra/naive_test_env.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```
Comment (malnick) [inline packages/dcos-integration-test/extra/naive_test_env.sh]: Can we have some comments up here on what this file does?

**Comment by spahl** [comment]
@mellenburg can you rename this to something else? Like run_in_cluster.sh or something?

**Comment by mellenburg** [comment]
@spahl Sure, I will also add a comment as @malnick pointed out as well.

However, this script doesnt run anything, just sets the env and leaves the user to run py.test (very intentional). So how about test_env.sh?

**Comment by spahl** [comment]
Oh I see, sounds good.

**Comment by spahl** [comment]
:ship:

**Pull request merged**